### PR TITLE
 NO-REF--Modify-fulfill-test-to-snake-case

### PR DESCRIPTION
**Pull request by mitri-slory**

### CODE REVIEW

File: tests/unit/processes/file/test_fulfill_manifest_process.py [hunk 1]
```diff
@@ -48,10 +48,9 @@ def test_fetch_and_update_manifests(self, testProcess, mocker):
             update_metadata_object=mocker.DEFAULT
         )
 
-        mockPrefix = mocker.MagicMock(prefix='testPrefix')
-        mockTimeStamp = mocker.MagicMock(timeStamp='testTimeStamp')
+        mockTimeStamp = mocker.MagicMock(timeStamp='test_timestamp')
```
Comment (kylevillegas93) [inline tests/unit/processes/file/test_fulfill_manifest_process.py:51]: Nice - good to get rid of the extra mock here!

File: tests/unit/processes/file/test_fulfill_manifest_process.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def __init__(self):
                 self.batchSize = mocker.MagicMock(batchSize='testBatchSize')
                 self.host = mocker.MagicMock(host='testHost')
                 self.port = mocker.MagicMock(port='testPort')
-                self.prefix = 'testPrefix'
+                self.prefix = 'test_prefix'
```
Comment (kylevillegas93) [inline tests/unit/processes/file/test_fulfill_manifest_process.py:27]: Actual variable values don't necessarily need to be in snake case!

**Comment by kylevillegas93** [comment]
Here are the PEP8 guidelines - recommend updating these tests accordingly: 

In Python, snake_case is the convention for most names, as recommended by [PEP 8](https://peps.python.org/pep-0008/), the official Python style guide:

- Variables and functions: Use snake_case, e.g., my_variable, calculate_sum().
- Classes: Use PascalCase (also known as CapitalizedWords or UpperCamelCase), e.g., MyClass.
- Constants: Use ALL_CAPS, e.g., MAX_LIMIT.


Adhering to snake_case in Python makes your code more readable and consistent with the broader Python ecosystem.

File: tests/unit/processes/file/test_fulfill_manifest_process.py (hunk 2 above)
Comment (mitri-slory) [reply tests/unit/processes/file/test_fulfill_manifest_process.py:27]: Oh ok thanks for the clarification. I'll change it back to camel case just to be consistent with the rest of the variable values in the test.

File: tests/unit/processes/file/test_fulfill_manifest_process.py [hunk 3]
```diff
@@ -30,28 +30,26 @@ def __init__(self):
...
 
-        mockPrefix = mocker.MagicMock(prefix='testPrefix')
-        mockTimeStamp = mocker.MagicMock(timeStamp='testTimeStamp')
+        mock_timestamp = mocker.MagicMock(time_stamp='test_timestamp')
         
-        testProcess.fetch_and_update_manifests(mockTimeStamp)
+        test_process.fetch_and_update_manifests(mock_timestamp)
 
-        processMocks['load_batches'].assert_called_once_with('testPrefix','test_aws_bucket')
+        process_mocks['load_batches'].assert_called_once_with('testPrefix','test_aws_bucket')
```
Comment (kylevillegas93) [inline tests/unit/processes/file/test_fulfill_manifest_process.py:55]: nit: newline!

**Review by kylevillegas93 — APPROVED** [review]
Ship it!

File: tests/unit/processes/file/test_fulfill_manifest_process.py (hunk 3 above)
Comment (mitri-slory) [reply tests/unit/processes/file/test_fulfill_manifest_process.py:55]: Line 55 is the last line in the file and it comes after line 54 as seen above, so a newline is already in place.

**Pull request merged**